kinto/core/authorization.py

```diff
@@ -176,7 +176,11 @@ def check_permission(self, principals, bound_perms):
         if not bound_perms:
             bound_perms = [(self.resource_name, self.required_permission)]
         for (_, permission) in bound_perms:
-            setting = "{}_{}_principals".format(self.resource_name, permission)
+            # With Kinto inheritance tree, we can have: `permission = "record:create"`
+            resource_name = self.resource_name
```

Comment:
Are you sure it is still better like that that with the first commit?

Reply:
https://github.com/Kinto/kinto/pull/1814/commits/fda8c13690cd2700371d00d488e00149fd5efd6a#diff-77d5feaac7fd094b43cdca245fac1ef4R179

Reply:
Now that you say it... one or the other I'm fine